great-idea/services.html

```diff
@@ -15,93 +15,159 @@
 </head>
 
 <body>
-
-
-Services
-Product
-Vision
-Features
-About
-Contact
+  <div class="container">
+      <nav>
+        <a href="Services">Services</a>
+        <a href="Product">Product</a>
+        <a href="Vision">Vision</a>
+        <a href="Features">Features</a>
+        <a href="About">About</a>
+        <a href="Contact">Contact</a>
 
 <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
always give your users a way back to the homepage without clicking the back button, a good way to accomplish this is add an href link on the logo img!